src/components/Pages/LoginPage/LoginPage.tsx

```diff
@@ -41,8 +42,14 @@ export const LoginPage = () => {
           setShowConnectionModal(true)
           setConnectionModalState(ConnectionModalState.CONNECTING_WALLET)
           const connectionData = await connectToProvider(connectionType)
-          setConnectionModalState(ConnectionModalState.WAITING_FOR_SIGNATURE)
-          await getSignature(connectionData.account?.toLowerCase() ?? '', connectionData.provider)
+
+          // The requests sign in flow for the desktop app has a different identity.
+          // There is no need to create one here if the user is coming from the requests page.
+          if (searchParams.get('fromRequests') !== 'true') {
+            setConnectionModalState(ConnectionModalState.WAITING_FOR_SIGNATURE)
+            await getSignature(connectionData.account?.toLowerCase() ?? '', connectionData.provider)
```

Comment:
If we don't get the signature here and store it, the user won't be logged in for other applications. Is that an issue?

Reply:
The signature is for the ephemeral account that is stored in the desktop client. That same signature will not be usable for the identity stored in the auth dapp.

This change just prevents the user from having to sign both the identity for the auth dapp and the identity for the desktop client.